src/components/binding/binding-renderer.js

```diff
@@ -186,7 +192,8 @@ genotet.BindingRenderer.prototype.init = function() {
  * @private
  */
 genotet.BindingRenderer.prototype.getBindingRanges_ = function() {
-  if (this.data.overviewRangeChanged) {
+  if (this.data.overviewRangeChanged || !this.scaleSet_) {
```

Comment:
It is my fault that I didn't set this.scaleSet_ back to false again. However, I haven't figured out that how to prevent that overwriting using overviewRangeChanged. Because updateRanges_() executes independently and asynchronously in each loadFullTrack() when load multiple tracks. Do you have any idea?